WordPress/src/main/java/org/wordpress/android/ui/media/MediaAddFragment.java

```diff
@@ -102,6 +118,9 @@ public void onReceive(Context context, Intent intent) {
                     ToastUtils.showToast(context, errorMessage, ToastUtils.Duration.SHORT);
                 }
                 mCallback.onMediaAdded(mediaId);
+            } else {
```

Comment:
In case other receivers are registered we should check for CONNECTIVITY_ACTION here. The intent is also broadcast when a network connection is lost. Maybe the network connection should be verified before calling onResume.